migrations/install/20180220023208_create_files_table.php

```diff
@@ -169,6 +174,16 @@ public function change()
                 'width' => 'half',
                 'required' => 1
             ],
+            [
+                'collection' => 'directus_files',
+                'field' => 'filename_download',
+                'type' => \Directus\Database\Schema\DataTypes::TYPE_STRING,
+                'interface' => 'text-input',
+                'locked' => 1,
+                'readonly' => 1,
```

Comment:
This shouldn't be readonly. The user is able to change the `filename_download` field at will to control how the file will be saved on download